src/status_im/utils/config.cljs

```diff
@@ -143,6 +143,7 @@
 
 (def wallet-connect-project-id "87815d72a81d739d2a7ce15c2cfdefb3")
 
+;; TODO moved to status-im2.common.constants
```

Comment:
can you create an issue for this please? 🙏 

Also as it's relevant, let's vote: 
https://notes.status.im/s/YzN7Ap_ia

Reply:
you have a TODO here, that implies there is a needed action to be done. 
If the removal of status-im covers this then we don't need this TODO comment and so you can remove it

Reply:
yeah its not TODO, its just a comment